java/blade/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+server.port=3000
```

Comment:
I've tried that, but seems that `blade` listen to `http://127.0.0.1:9000`
~~~java
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : app.env          » default
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : app.pid          » 6
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : app.devMode      » true
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : jdk.version      » 1.8.0_212
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : user.dir         » /usr/src/app
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : java.io.tmpdir   » /tmp
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : user.timezone    » GMT
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : file.encoding    » UTF-8
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : app.classpath    » /usr/src/app/target/benchmark-0.0.1-jar-with-dependencies.jar

                                         __, _,   _, __, __,
                                         |_) |   /_\ | \ |_
                                         |_) | , | | |_/ |
                                         ~   ~~~ ~ ~ ~   ~~~
                                     :: Blade :: (v2.0.15.RELEASE) 

2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]            c.b.m.r.RouteMatcher : » Add route  POST    /user
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]            c.b.m.r.RouteMatcher : » Add route  GET     /
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]            c.b.m.r.RouteMatcher : » Add route  GET     /user/:id
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : » Register bean: [com.blade.Environment@6f51b44c, com.example.UserController@32baeab5]
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : » Use NioEventLoopGroup
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : » Blade initialize successfully, Time elapsed: 366 ms
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : » Blade start with  0.0.0.0:9000 
2020/05/03 13:05:15  INFO [          _(:3」∠)_ ]             c.b.s.n.NettyServer : » Open browser access http://127.0.0.1:9000 ⚡
~~~

Reply:
I did a `mvn clean` and re-did the build and it worked fine. Alternatively just `rm -rf target/`.